build.c

```diff
@@ -495,6 +521,15 @@ jobwork(struct job *j)
 	return false;
 }
 
+#if HAVE_TIOCGWINSZ
+static void
+sig_handler(int sig)
```

Comment:
Without checking if sig == SIGWINCH there's a -Wunused-parameter warning and I didn't want to add a warning but I also didn't want to add -Wno-unused-parameter to the compilation flags, any other ideas for how to avoid it?

Reply:
I pushed a set of changes to deal with your suggestions, currently has the -Wunused-parameter I mentioned, so that'll need to be dealt with.

Reply:
I think a common way to deal with this warning is

```c
(void)sig;
```

Reply:
Thanks!